cudnn.spec

```diff
@@ -14,13 +13,13 @@
 # cuDNN archive base name and unpacked name
 %define archive cudnn-linux-%{nvarch}-%{realversion}_cuda%{cudaver}-archive
 
-Source: https://developer.download.nvidia.com/compute/redist/cudnn/v%{cudnnver_maj}/local_installers/%{cudaver}/%{archive}.tar.xz
+Source: https://developer.download.nvidia.com/compute/cudnn/redist/cudnn/linux-%{nvarch}/%{archive}.tar.xz
```

Comment:
@smuzaffar, the location of the cuDNN archives has changed since version 8.4.4.
The old location (under https://developer.download.nvidia.com/compute/redist/cudnn/) has only the RPM and DEB packages.
The new location (under https://developer.download.nvidia.com/compute/cudnn/redist/cudnn/) has the .tar.xz archives.